### PR TITLE
Update `@sats-connect/core` version

### PR DESCRIPTION
**Pull request by aryzing**
- Updates sats-connect-core to latest version
- Updates example app to include
  - getBtcBalance
  - getRunesBalance
  - getInscriptions
  - Use updated connect / permissions request

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -26,10 +28,41 @@ function App() {
...
     if (response.status === 'success') {
       setAddressInfo(response.result);
-      console.log(response.result);
     }
   };
 
+  const onConnect2 = useCallback(async () => {
+    const res = await Wallet.request('wallet_requestPermissions', undefined);
+    if (res.status === 'error') {
+      console.error('Error connecting ot wallet, details in terminal.');
```
Comment (teebszet) [inline example/src/App.tsx]:
```suggestion
      console.error('Error connecting to wallet, details in terminal.');
```

**Review by teebszet — DISMISSED** [review]
nice! ⚠️ needs core version released

**Comment by teebszet** [comment]
nit: Connect2 could do with a gap and better name.. Connect (>=v2.5.0)? would we ultimately want to provide a Connect button which tries connect2 and degrades gracefully to connect1?

image

**Review by teebszet — DISMISSED** [review]

**Comment by aryzing** [comment]
Good nit, fixed
image

File: example/src/components/SendBtc/index.tsx [hunk 2]
```diff
@@ -42,14 +42,14 @@ const SendBtc = ({ network }: Props) => {
       <h3>Send BTC</h3>
       {!txnId && (
         <>
-          <p>
+          <div>
```
Comment (aryzing) [inline example/src/components/SendBtc/index.tsx:45]: Changed b/c `p`s can't have `div`s as children

File: src/index.ts [hunk 3]
```diff
@@ -52,6 +52,7 @@ class Wallet {
   }
 
   public async disconnect() {
+    await this.request('wallet_renouncePermissions', undefined);
```
Comment (aryzing) [inline src/index.ts:55]: "Disconnecting" here, works cc @m-aboelenein

File: example/src/App.tsx [hunk 4]
```diff
@@ -9,27 +9,62 @@ import {
...
+import GetRunesBalance from './components/GetRunesBalance';
+import { ConnectButtonsContainer } from './App.styles';
 
 function App() {
   const [network, setNetwork] = useLocalStorage<BitcoinNetworkType>(
     'network',
     BitcoinNetworkType.Mainnet
   );
-  const [addressInfo, setAddressInfo] = useLocalStorage<Address[]>('addresses', []);
+  const [addressInfo, setAddressInfo] = useState<Address[]>([]);
```
Comment (m-aboelenein) [inline example/src/App.tsx]: storing the data in localStorage is necessary to persist the data across refreshes

Comment (aryzing) [reply example/src/App.tsx]: Thanks, forgot to change it back, had to clear it a few times while debugging

**Review by Christine-Pinto — APPROVED** [review]

**Pull request merged**